packages/models/src/errors.ts

```diff
@@ -85,7 +86,7 @@ export function makeApiProblemBuilder<T extends string>(errors: {
   [K in T]: string;
 }): MakeApiProblemFn<T> {
   const allErrors = { ...errorCodes, ...errors };
-  return (error, httpMapper, logger) => {
+  return (error, httpMapper, logger, logMessage) => {
```

Comment:
`logMessage` seems too much generic, Could be more expressive using parameter name like `operationalMsg` ?

```suggestion
  return (error, httpMapper, logger, operationalMsg) => {
```